src/index.js

```diff
@@ -70,17 +70,16 @@ export default async function loader(content, sourceMap, meta) {
   );
 
   if (useSourceMap) {
-    processOptions.map = { inline: false, annotation: false };
+    if (
+      typeof processOptions.map === 'undefined' ||
+      typeof processOptions.map === 'boolean'
```

Comment:
`processOptions.map` means as the same `postcss(plugins).process(content, { from, map: true })`, there is not docs for this, but it should generate inline map as I remember

Reply:
Actually it should generate external sourcemaps

https://github.com/postcss/postcss/blob/master/docs/source-maps.md

Reply:
>To generate a new source map with the default options, simply set map: true. This will generate an inline source map that contains the source content. If you don’t want the map inlined, you can set map.inline: false.